x-pack/plugins/uptime/public/components/functional/monitor_list_actions_popover.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiButtonIcon,
+  EuiFlexGrid,
+  EuiFlexItem,
+  EuiPopover,
+  EuiText,
+  EuiToolTip,
+} from '@elastic/eui';
+import React, { useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { get } from 'lodash';
+import { LatestMonitor } from '../../../common/graphql/types';
+import { IntegrationLink } from './integration_link';
+
+interface MonitorListActionsPopoverProps {
+  basePath: string;
+  dateRangeStart: string;
+  dateRangeEnd: string;
+  monitor: LatestMonitor;
+}
+
+export const MonitorListActionsPopover = ({
+  basePath,
+  dateRangeStart,
+  dateRangeEnd,
+  monitor,
+  monitor: { ping },
+}: MonitorListActionsPopoverProps) => {
+  const popoverId = `${monitor.id.key}_popover`;
+  const [popoverIsVisible, setPopoverIsVisible] = useState<boolean>(false);
+  const domain = get(ping, 'url.domain', '');
+  return (
+    <EuiPopover
+      button={
+        <EuiButtonIcon
+          aria-label={i18n.translate(
+            'xpack.uptime.monitorList.observabilityIntegrationsColumn.popoverIconButton.ariaLabel',
+            {
+              defaultMessage: 'Opens integrations popover for monitor with url {monitorUrl}',
+              description:
+                'A message explaining that this button opens a popover with links to other apps for a given monitor',
+              values: { monitorUrl: monitor.id.url },
+            }
+          )}
+          color="subdued"
+          iconType="boxesHorizontal"
+          onClick={() => setPopoverIsVisible(true)}
+        />
+      }
+      closePopover={() => setPopoverIsVisible(false)}
+      id={popoverId}
+      isOpen={popoverIsVisible}
+    >
+      <EuiFlexGrid>
+        <EuiFlexItem>
+          <EuiToolTip
+            content={i18n.translate(
+              'xpack.uptime.monitorList.observabilityIntegrationsColumn.apmIntegrationLink.tooltip',
+              {
+                defaultMessage: 'Click here to check APM for the domain "{domain}".',
```

Comment:
Can we make the message clickable as well?

Reply:
Add in https://github.com/elastic/kibana/pull/34892/commits/169a347c74ebd9d754f0945349a28437fdacc590.